docs/gfx.md

```diff
@@ -0,0 +1,124 @@
+ZJS API for GFX
+===============
+
+* [Introduction](#introduction)
+* [Web IDL](#web-idl)
+* [API Documentation](#api-documentation)
+* [Sample Apps](#sample-apps)
+
+Introduction
+------------
+The GFX module provides a generic way to create pixel buffers, which can then
+be displayed on a display of some kind.  A Javascript method for initializing
+the screen, and drawing a data buffer are required to use it.
+See module/ST7735.js and samples/SPI_Screen.js for an example.
+
+Web IDL
+-------
+This IDL provides an overview of the interface; see below for documentation of
+specific API functions.
+
+```javascript
+// require returns a GFX object
+// var gfx = require('gfx');
+
+[NoInterfaceObject]
+interface GFX {
+    GFXObj init(screen width, screen height, init screen function,
```

Comment:
JS uses Object, so I would write it out. but maybe GfxContext makes more sense, especailly since canvas uses Context